be/src/exec/pipeline/pipeline_builder.cpp

```diff
@@ -6,34 +6,67 @@
 
 namespace starrocks::pipeline {
 
+OpFactories PipelineBuilderContext::maybe_interpolate_local_broadcast_exchange(OpFactories& pred_operators,
+                                                                               int num_receivers) {
+    if (num_receivers == 1) {
+        return maybe_interpolate_local_passthrough_exchange(pred_operators);
+    }
+
+    auto pseudo_plan_node_id = next_pseudo_plan_node_id();
+    auto mem_mgr =
+            std::make_shared<LocalExchangeMemoryManager>(config::vector_chunk_size * num_receivers * num_receivers);
```

Comment:
```suggestion
            std::make_shared<LocalExchangeMemoryManager>(config::vector_chunk_size  * num_receivers);
```

Reply:
@kangkaisen  When every local exchange source receives a broadcast chunk, it will add `chunk.size()` to `mem_mgr`. Therefore, each broadcast chunk will add `num_receivers ` times. 
If we set rows limit as `config::vector_chunk_size * num_receivers`, only one local broadcast exchange sink can push chunk, other sinker will see the exchanger is full.